content/blog/prototyping/index.md

```diff
@@ -0,0 +1,767 @@
+
+I find it quite useful for quickly jotting down a bunch of functions or types to test out a design idea. 
+
+### Use `unwrap` Liberally
+
+It's okay to use `unwrap` in the early stages of your project.
+An explicit `unwrap` is like a stop sign that tells you "here's something you need to fix later."
+You can easily grep for `unwrap` and replace it with proper error handling later when you polish your code.
+This way, you get the best of both worlds: quick iteration cycles and a clear path to robust error handling.
+
+```rust
+use std::fs;
+use std::path::PathBuf;
+
+fn main() {
+    // Quick and dirty path handling during prototyping
+    let home = std::env::var("HOME").unwrap();
+    let config_path = PathBuf::from(home).join(".config").join("myapp");
+    
+    // Create config directory if it doesn't exist
+    fs::create_dir_all(&config_path).unwrap();
+    
+    // Read the config file, defaulting to empty string if it doesn't exist
+    let config_file = config_path.join("config.json");
+    let config_content = fs::read_to_string(&config_file)
+        .unwrap_or_default();
+    
+    // Parse the JSON config
+    let config: serde_json::Value = if !config_content.is_empty() {
+        serde_json::from_str(&config_content).unwrap()
+    } else {
+        serde_json::json!({})
+    };
+    
+    println!("Loaded config: {:?}", config);
+}
+```
+
+See all those unwraps?
+To more experienced Rustaceans, they stand out like a sore thumb -- and that's a good thing!
+
+Compare that to languages like JavaScript which can throw exceptions your way at any time.
+It's much harder to ensure that you handle all the edge-cases correctly.
+At the very least, it costs time. Time you could spend on more important things.
+
+While prototyping with Rust, you can safely ignore error handling and focus on
+the happy path without losing track of improvement areas. 
+
+### Use `bacon` for quick feedback cycles
+
+Rust is not a scripting language; there is a compile step!
+
+However, for small projects, the compile times are negligible.
+Unfortunately, you have to manually run `cargo check` every time you make a change 
+or use [rust-analyzer](https://rust-analyzer.github.io/) in your editor to get instant feedback.
+
+To fill the gap, you can use external tools like [`bacon`](https://github.com/Canop/bacon) which automatically recompiles and runs your code whenever you make a change.
+This way, you can get *almost* the same experience as with a REPL in, say, Python or Ruby.
+
+The setup is simple:
+
+```sh
+# Install bacon
+cargo install --locked bacon
+
+# Run bacon in your project directory
+bacon
+```
+
+And just like that, you can get some pretty compilation output alongside your code editor.
+
+![bacon](bacon.png)
+
+Oh, and in case you were wondering, `cargo-watch` was another popular tool for
+this purpose, but it's since been [deprecated](https://github.com/watchexec/cargo-watch#maintenance).
+
+### `cargo-script` is awesome
```

Comment:
This is going to read weird, if you apply capitalization consistently. Maybe a good argument for using sentence case for titles at this level, and use title case only for the `## Titles`.

Either way, this is going to be my last comment on this, even though you keep mixing styles further below too :smile: